Samples/PushNotifications/README.md

```diff
@@ -0,0 +1,57 @@
+---
+name: "Push Notifications Sample" 
+page_type: sample
+languages:
+- cpp
+products: 
+- windows-app-sdk
+description: "Shows how to use the windows app SDK Push Notifications APIs from an unpackaged app"
+urlFragment: PushNotifications
+---
+# Push Notifications Sample
+This sample demonstrates 
+1. How an unpackaged app can use the WindowsAppSDK to request a channel URI
+    * The Channel URI uniquely identifies the app for a current user on a specific device, allowing other apps and servers to send notifications to this app.
+2.	How to use the WindowsAppSDK to register an app for foreground and background activation.
+The Code
+When it is launched normally (by the users, or from the debugger), the sample requests a channeluri and displays it, then waits for notifications.
+    * This allows to copy the channeluri and sent notifications
+When it is activated from a push notification, the sample displays the notification.
+    * It doesn’t register for foreground activation of perform any other actions.
+       * Because background activation is meant to let app perform only small tasks in order to preserve battery life.
+## Requirements
+Cpp sample requires a ... like Postman or fiddler to send notifications.
```

Comment:
`requires an app like`?

Reply:
Maybe publisher app

Reply:
I think what I was looking for was "web API tool"